skiko/src/androidMain/kotlin/org/jetbrains/skiko/Convertors.android.kt

```diff
@@ -48,21 +48,37 @@ private fun toSkikoModifiers(event: KeyEvent): SkikoInputModifiers {
     return SkikoInputModifiers(result)
 }
 
-fun toSkikoTouchEvent(event: MotionEvent, index: Int, density: Float): SkikoTouchEvent {
-    return SkikoTouchEvent(
-        x = (event.getX(index) / density).toDouble(),
-        y = (event.getY(index) / density).toDouble(),
-        timestamp = event.getEventTime(),
-        kind = toSkikoTouchEventKind(event),
-        platform = event
-    )
-}
+fun toSkikoPointerEvent(event: MotionEvent, density: Float): SkikoPointerEvent {
+    val upIndex = when (event.action) {
+        MotionEvent.ACTION_UP -> 0
+        MotionEvent.ACTION_POINTER_UP -> event.actionIndex
+        else -> -1
+    }
 
-fun toSkikoGestureEvent(event: MotionEvent, density: Float): SkikoGestureEvent {
-    return SkikoGestureEvent(
-        x = (event.x / density).toDouble(),
-        y = (event.y / density).toDouble(),
-        kind = toSkikoGestureEventKind(event),
+    val pointers = (0 until event.pointerCount).map {
+        SkikoPointer(
+            x = event.getX(it).toDouble() / density,
+            y = event.getY(it).toDouble() / density,
+            // Same as in Jetpack Compose for Android
+            // https://github.com/androidx/androidx/blob/58597f0eba31b89f57b6605b7ed4977cd48ed38d/compose/ui/ui/src/androidMain/kotlin/androidx/compose/ui/input/pointer/MotionEventAdapter.android.kt#L126
+            // (we don't support Mouse for Android yet, so we don't check hover)
```

Comment:
I'd avoid personal pronouns in comments
```suggestion
            // (Mouse isn't supported for Android yet, so hover cannot be checked)
```

Reply:
Done